contributors/guide/pull-requests.md

```diff
@@ -280,26 +277,41 @@ Each commit should have a good title line (<70 characters) and include an additi
 
  Don't squash when there are independent changes layered to achieve a single goal. For instance, writing a code munger could be one commit, applying it could be another, and adding a precommit check could be a third. One could argue they should be separate pull requests, but there's really no way to test/review the munger without seeing it applied, and there needs to be a precommit check to ensure the munged output doesn't immediately get out of date.
 
-A commit, as much as possible, should be a single logical change.
-
-## 7. KISS, YAGNI, MVP, etc.
+## 7. Commit Message Guidelines
+ 
+- Each commit should have a good title line (< 50 characters) and can include an additional description paragraph in the PR Description describing the change in more detail.
+- Wrap the body of the commit message at 72 characters and use the body to explain **What and Why** is the change and **How** it has been done.
+- It is better if the Pull Request follow the `Semantic PR Commit Guidelines` which is `{Type}: {Subject}`. **Types** can be classified as :
```

Comment:
Would it generally make sense to stick to `Type == kind`? If yes, we could link the kind label to this sort of typing guideline.

Reply:
I think that's a great idea. Even if there is more than one kind applied to it, can defer to what it primarily is.

Reply:
>  If yes, we could link the kind label to this sort of typing guideline.

Sounds great! I don't think currently I have the right to create labels right now.

Reply:
We wouldn't create new labels, but use the ones we currently use. For reference, here is our PR template for the k/k repo:
https://raw.githubusercontent.com/kubernetes/kubernetes/master/.github/PULL_REQUEST_TEMPLATE.md

In there we have a block:
```

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake
```

That would add some context to the commit without having to reference the labels associated with the PR.

Reply:
I think it's also reasonable to use "{area}" in addition to kind.

Reply:
Agree 👍